src/app/services/csv.service.ts

```diff
@@ -0,0 +1,96 @@
+import { Injectable } from '@angular/core';
+
+import { FirestoreScholar, PaymentMethods } from '../_models/scholar';
+import { UserService } from './user/user.service';
+
+export type Header = {
+  title: string;
+  fieldName: keyof FirestoreScholar;
+  replacer: (key: string, value: any) => string;
+};
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CsvService {
+  private _scholars: FirestoreScholar[];
+
+  constructor(private userService: UserService) {
+    this.userService.getScholars().subscribe((scholars) => {
+      this._scholars = scholars;
+    });
+  }
+
+  export() {
+    const headers: Header[] = [
+      {
+        title: 'Ronin Address',
+        fieldName: 'roninAddress',
+        replacer: (key, value: string) => (value === null ? '' : value),
+      },
+      {
+        title: 'Name',
+        fieldName: 'name',
+        replacer: (key, value: string) => (value === null ? '' : value),
+      },
+      {
+        title: 'Group',
+        fieldName: 'group',
+        replacer: (key, value: string) => (value === null ? '' : value),
+      },
+      {
+        title: 'Email',
+        fieldName: 'email',
+        replacer: (key, value: string) => (value === null ? '' : value),
+      },
+      {
+        title: 'Use own payscale',
+        fieldName: 'useOwnPayShare',
+        replacer: (key, value: boolean) => (value === true ? 'true' : 'false'),
+      },
+      {
+        title: "Manager's share",
+        fieldName: 'managerShare',
+        replacer: (key, value) => value.toString(),
```

Comment:
can we return just value instead of converting to a string? because then in the csv file we get something like this 
![image](https://user-images.githubusercontent.com/34351874/135407891-7cf63eab-9ff5-4693-bcbe-acd27c828163.png)
which then the importing does not like 
![image](https://user-images.githubusercontent.com/34351874/135407993-bc211a3a-44b1-442a-8c99-29f62953b3c6.png)

Reply:
Fixed!